zbarimg/zbarimg.c

```diff
@@ -83,18 +83,19 @@ static const char *note_usage = N_(
     "scan and decode bar codes from one or more image files\n"
     "\n"
     "options:\n"
-    "    -h, --help      display this help text\n"
-    "    --version       display version information and exit\n"
-    "    -q, --quiet     minimal output, only print decoded symbol data\n"
-    "    -v, --verbose   increase debug output level\n"
-    "    --verbose=N     set specific debug output level\n"
-    "    -d, --display   enable display of following images to the screen\n"
-    "    -D, --nodisplay disable display of following images (default)\n"
-    "    --xml, --noxml  enable/disable XML output format\n"
-    "    --raw           output decoded symbol data without converting charsets\n"
-    "    -1, --oneshot   exit after scanning one bar code\n"
+    "    -h, --help       display this help text\n"
+    "    --version        display version information and exit\n"
+    "    -q, --quiet      minimal output, only print decoded symbol data\n"
+    "    -v, --verbose    increase debug output level\n"
+    "    --verbose=N      set specific debug output level\n"
+    "    -d, --display    enable display of following images to the screen\n"
+    "    -D, --nodisplay  disable display of following images (default)\n"
+    "    --xml, --noxml   enable/disable XML output format\n"
+    "    --raw            output decoded symbol data without converting charsets\n"
+    "    --rawnodelimiter same as --raw, but without newlines between symbol data\n"
+    "    -1, --oneshot    exit after scanning one bar code\n"
     "    -S<CONFIG>[=<VALUE>], --set <CONFIG>[=<VALUE>]\n"
-    "                    set decoder/scanner <CONFIG> to <VALUE> (or 1)\n"
+    "                     set decoder/scanner <CONFIG> to <VALUE> (or 1)\n"
     // FIXME overlay level
```

Comment:
Same above at line 98. Also, I would expect just one line added there with
--rawnodedelimiter option, instead of touching everything.